rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -154,11 +154,15 @@ def set_match_state(self, match_state: msg.MatchState):
         self.match_state = match_state
 
     def debug_callback(self, play: stp.play.IPlay, tactics: list):
+        """
+        Publishes the string that shows up in the behavior tree in the Soccer UI.
+        """
         debug_text = ""
-        debug_text += f"{type(play).__name__}({type(self.play_selector.curr_situation).__name__})\n"
+        debug_text += f"Play: {play}\n\n"
+        debug_text += f"Play Selector: {self.play_selector}\n"
```

Comment:
Print the type of the self.play_selector instead of self.play_selector. It is currently printing NoneType.

Reply:
not quite, should be printing the type of self._current_situation for desired behavior. I fixed just now

Reply:
That makes sense thank you!